src/plugins/hetero/executable_network.cpp

```diff
@@ -723,7 +735,8 @@ void HeteroExecutableNetwork::Export(std::ostream& heteroModel) {
     heteroModel << std::endl;
 
     for (auto&& subnetwork : _networks) {
-        if (_heteroPlugin->GetCore()->DeviceSupportsModelCaching(subnetwork._device)) {
+        if (std::dynamic_pointer_cast<InferenceEngine::ICore>(_plugin->get_core())
```

Comment:
In theory `std::dynamic_pointer_cast<InferenceEngine::ICore>(_plugin->get_core())` can be nullptr...